test/e2e/azure.go

```diff
@@ -1,3 +1,4 @@
+//go:build e2e
```

Comment:
This build tag was added because you're using a newer version of go than currently used to build this operator.

Reply:
i use `go1.16.4`  , but i will remove it now, thx for catching this 🙏🏽